Ginger/GingerCoreNET/Run/RunListenerLib/LiteDBRepository.cs

```diff
@@ -616,56 +615,36 @@ public override string GetLogFolder(string folder)
             return string.Empty;
         }
 
-        public override async Task<bool> SendExecutionLogToCentralDBAsync(LiteDB.ObjectId runsetId, Guid executionId, eDeleteLocalDataOnPublish deleteLocalData)
+        public void DeleteLiteDbAndScreenShotData(LiteDB.ObjectId runsetId, Guid executionId)
         {
-            //Get the latest execution details from LiteDB
-            LiteDbManager dbManager = new LiteDbManager(new ExecutionLoggerHelper().GetLoggerDirectory(WorkSpace.Instance.Solution.LoggerConfigurations.CalculatedLoggerFolder));
-            LiteDbRunSet liteDbRunSet = dbManager.GetLatestExecutionRunsetData(runsetId?.ToString());
+            LiteDbRunSet liteDbRunSet = this.liteDbManager.GetLatestExecutionRunsetData(runsetId?.ToString());      
             List<string> screenshotList = PopulateMissingFieldsAndGetScreenshotsList(liteDbRunSet, executionId);
+            
+            try
+            {
 
-            AccountReportApiHandler centralExecutionLogger = new AccountReportApiHandler(WorkSpace.Instance.Solution.LoggerConfigurations.CentralLoggerEndPointUrl);
-
-            //Map the data to AccountReportRunset Object
-            AccountReportRunSet accountReportRunSet = centralExecutionLogger.MapDataToAccountReportObject(liteDbRunSet);
-            SetExecutionId(accountReportRunSet, executionId);
-            accountReportRunSet.EntityId = WorkSpace.Instance.RunsetExecutor.RunSetConfig.Guid;
-            accountReportRunSet.GingerSolutionGuid = WorkSpace.Instance.Solution.Guid;
-
-            //Publish the Data and screenshots to Central DB
-            await centralExecutionLogger.SendRunsetExecutionDataToCentralDBAsync(accountReportRunSet);
-            await centralExecutionLogger.SendScreenShotsToCentralDBAsync(executionId, screenshotList);
+                this.liteDbManager.DeleteDocumentByLiteDbRunSet(liteDbRunSet);
+            }
+            catch (Exception ex)
+            {
+                Reporter.ToLog(eLogLevel.ERROR, "Error when deleting local LiteDB data after Publis", ex);
+            }
 
 
-            //Delete local data if configured
-            if (deleteLocalData == eDeleteLocalDataOnPublish.Yes)
+            foreach (string screenshot in screenshotList)
             {
                 try
                 {
-                    dbManager.DeleteDocumentByLiteDbRunSet(liteDbRunSet);
```

Comment:
Deleting data from local db is not needed?

Reply:
I have created a new function for deleting the data from the db in LiteDbRepository as the above function had some extra functions that were not needed.